### ISSUE TITLE
[MARKUP]: GIBCT - Revisit the provider detail pages to refactor, remove extra ARIA attributes

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#8392]
**Feedback framework**

- **⚠️ Should** if the feedback is best practice

## Description

The provider detail view no longer throws an axe error because the team has refactored the unordered list of accordions. While discussing the issue with the development team, we discovered a need for `aria-labels` on each `<li>` to prevent unexpected screen reader behavior. This should be revisited soon, and all extra ARIA markup removed from the page. Specifically look at at `aria-live`, `aria-label` attributes, as they're not needed with the current implementation.

## Related Issues
* https://github.com/department-of-veterans-affairs/va.gov-team/issues/7727

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Acceptance Criteria

- [ ] Screen readers do not announce live region changes. Accordions should be able to be opened and closed, with screen readers announcing the button is "expanded" or "collapsed", but not start reading the elements inside them.
- [ ] No axe violations appear
- [ ] Semantic HTML validates a checker like the W3C HTML5 validator

## Environment

* https://staging.va.gov/gi-bill-comparison-tool/profile/14900416 is a good example page, but any provider detail view will work

## WCAG or Vendor Guidance (optional)

* [Info and Relationships: Understanding SC 1.3.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-programmatic.html)

**Comment by dmccurdybah** [comment]
PR: https://github.com/department-of-veterans-affairs/vets-website/pull/12362

**Comment by sonntag-adam** [comment]
@1Copenut in staging.

**Comment by 1Copenut** [comment]
@sonntag-adam I checked this in Staging with Chrome + JAWS and Firefox + NVDA. Both worked as expected. Moving to close.
